src/repeated_game.jl

```diff
@@ -221,19 +221,23 @@ end
 
 Given a constraint w ∈ W, this finds the worst possible payoff for agent i.
 
-# Arugments 
+# Arugments
 
 - `rpd::RepGame2` : Two player repeated game.
 - `H::Array{Float64, 2}` : The subgradients used to approximate the value set.
 - `C::Array{Float64, 1}` : The array containing the hyperplane levels.
 - `i::Int` : The player of interest.
+- `lp_solver` : Allows users to choose a particular solver for linear
+   programming problems. Options include ClpSolver(), CbcSolver(),
+   GLPKSolverLP() and GurobiSolver(). By default, it choooses ClpSolver().
+
 
 # Returns
 
 - `out::Float64` : Worst possible payoff for player i.
 """
 function worst_value_i(rpd::RepGame2, H::Array{Float64, 2},
-                       C::Array{Float64, 1}, i::Int)
+                       C::Array{Float64, 1}, i::Int, lp_solver=ClpSolver())
```

Comment:
Add a type declaration. (Several places.)

Reply:
Should I add and use  `AbstractMathProgSolver` as in `linprog`?

Reply:
Yes. Thanks.